src-plugins/itkFilters/itkMorphologicalFiltersToolBox.cpp

```diff
@@ -85,6 +85,7 @@ itkMorphologicalFiltersToolBox::itkMorphologicalFiltersToolBox ( QWidget *parent
     QLabel * morphoFilterLabel = new QLabel ( tr ( "Kernel radius:" ) );
     QHBoxLayout * morphoFilterLayout = new QHBoxLayout;
     d->mmButton = new QRadioButton(tr("mm"), this);
+    d->mmButton->setObjectName("mm");
```

Comment:
I think mm is not descriptive enough as a name for an object. I prefer mmButton or something else.